src/services/FacilityService.ts

```diff
@@ -1,14 +1,28 @@
+import { Facility } from "../types/Facility";
+
 const KEYS = {
-    facilities: "facilities",
-  };
-  export function getAllFacilities() {
-    if (localStorage.getItem(KEYS.facilities) == null)
-      localStorage.setItem(KEYS.facilities, JSON.stringify([]));
-    return myPromise.then((value) => value);
-  }
-  
-  const myPromise = new Promise((resolve, reject) => {
+  facilities: "facilities",
+};
+export const getAllFacilities = async (): Promise<Facility[]> => {
+  const facilities: Facility[] = JSON.parse(
+    localStorage.getItem(KEYS.facilities) || "[]"
```

Comment:
if you are already setting this to empty array, why do u need to return empty array again at the end of the function?

Reply:
Right, Forget to remove that. I will change and do PR again. Thank youe